Corzbank.Api/Controllers/CardsController.cs

```diff
@@ -33,6 +33,15 @@ public async Task<IActionResult> GetCards()
             return Ok(result);
         }
 
+        [Route("user/{id}")]
```

Comment:
user**S**

Reply:
why users if I'll return items for specific user

Reply:
REST architecture. collection/specific
https://stackoverflow.com/a/29709867

Reply:
user**s**/{id}

Reply:
sure, thanks)